.env.example

```diff
@@ -1,10 +1,30 @@
-DATABASE_URL="mysql://root@127.0.0.1:3309/video-course-starter-kit"
+# Set this value to the canonical URL of your site
+
 NEXTAUTH_URL="http://localhost:3000"
```

Comment:
Can we move this to the bottom and note that this is only required for local development? (no need to set it in prod)

Reply:
the `NEXTAUTH_URL` I mean